models/ulmfit/__init__.py

```diff
@@ -1,17 +1,116 @@
-from typing import List, Dict
+from typing import List, Dict, Optional
+import pickle
+import tensorflow as tf
+
+from models.ulmfit.model import *
 
 from models import Embedding
 
-EMBEDDING_MODELS: List[Embedding] = [
-                    Embedding(name=u'umlfit',
-                              dimensions=300,
-                              corpus_size='570k human-generated English sentence pairs',
-                              vocabulary_size='230k',
-                              download_url='http://files.fast.ai/models/wt103/',
-                              format='.h5',
-                              architecture='cbow',
-                              trained_data='Stephen Merity’s Wikitext 103 dataset',
-                              language='en')
-                                ]
-
-EMBEDDING_MODELS: Dict[str, Embedding] = {embedding.name: embedding for embedding in EMBEDDING_MODELS}
\ No newline at end of file
+
+class Embeddings(object):
+    EMBEDDING_MODELS: List[Embedding] = [
+                        Embedding(name=u'umlfit',
+                                  dimensions=300,
+                                  corpus_size='570k human-generated English sentence pairs',
+                                  vocabulary_size='230k',
+                                  download_url='http://files.fast.ai/models/wt103/',
+                                  format='.h5',
```

Comment:
Actually after we download this file, we try to extract all files inside some folder, since we don't have any unzip feature for `.h5`.  Can you test if this is working ?